war/WEB-INF/applicationContext-service.xml

```diff
@@ -24,9 +24,6 @@
 	<bean id="loginService" class="mat.server.service.impl.LoginCredentialServiceImpl">	    
 	</bean>
 	
-	<bean id="featureFlag" class="mat.server.FeatureFlagServiceImpl">	    
-	</bean>
-	
```

Comment:
Good. It was registered 2 times. 1st in xml and the other one with the annotation.
Same issue we have for the new gwt servlets

Reply:
Yup